packages/util/src/referralSystem/index.ts

```diff
@@ -29,6 +29,14 @@ export class ReferralSystemModule {
     return this.getReferrer(this.root.account.pair.address);
   }
 
+  /**
+   * Returns the referrer subscription
+   * @param invitedUserId address of invited account
+   */
+  public subscribeOnReferrer(invitedUserId: string): Observable<string> {
+    return this.root.apiRx.query.referrals.referrers(invitedUserId) as unknown as Observable<string>;
```

Comment:
If you want convert `Codec `to `string`, you should transform it here:

`return this.root.apiRx.query.referrals.referrers(invitedUserId).pipe(map(codec => codec.toJSON() as null | string))`